pages/in/widget/wordpress/index.vue

```diff
@@ -0,0 +1,700 @@
+<!-- eslint-disable max-len -->
```

Comment:
Also would be nice if the two steps can be split into two components, even better is to refactor the existing two old widgets to share these components since they share a lot logic in common

Reply:
Renamed into iscn-ar as discussed